app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java

```diff
@@ -217,6 +222,28 @@ private void renderSessions(List<SessionViewModel> adjustedSessionViewModels) {
             binding.txtDate.setText(adjustedSessionViewModels.get(0).getFormattedDate());
             binding.txtDate.setVisibility(View.VISIBLE);
         }
+        scrollTo(extractTodayPos(getContext(), new Date(), adjustedSessionViewModels));
+    }
+
+    private int extractTodayPos(Context context, Date today, List<SessionViewModel> adjustedSessionViewModels) {
+        String todayString = DateUtil.getMonthDate(today, context);
+        return Stream.of(adjustedSessionViewModels)
+                .map(SessionViewModel::getFormattedDate)
+                .indexed()
+                .filter(pair -> todayString.equals(pair.getSecond()))
+                .findFirst()
+                .orElse(new IntPair<>(0, ""))
```

Comment:
Is it better to use `Stream`? In this case, just for loop seems easy. What do you think?

```java
for (int i = 0; i<= adjustedSessionViewModels.size(); i++) {
  SessionViewModel viewModel = adjustedSessionViewModels.get(i);
  if (todayString.equals(viewModel.getFormattedDate()) {
    return i;
  }
  return 0;
}
```